### PR TITLE
[Ready for Review] Exp-integration + personality results page

### PR DESCRIPTION
**Pull request by samanehsan**
## Purpose

Integrate ISP with exp-addons, so that it is visible in experimenter.
Users have the option to view a personality assessment after taking the survey.
## Changes
- Add `participate/index`route that embeds the exp-player (will replace the `survey` route)
- Login/authenticate with jamdb (remove StudyID field)
- Update the `participate` model hook to get the existing session instead of creating a new one.
- Add `participate/results` route that that renders the `results-page` component which determines the participant's personality traits from session.expData
- Remove `jam-auth` authenticator and authorizer & extend `jam-auth` component in exp-addons
## Reviewer notes:

Before this PR can get merged into develop...
- In addition to "high" and "low" trait descriptions, a "medium" category will also be added
- The scoring of the questions and trait score cut-offs will also change
- The survey will only show the overview, free-response, and card-sort sections. The rating-form and "thank-you" page have not been merged: https://github.com/CenterForOpenScience/exp-addons/pull/99

### CODE REVIEW

**Comment by samanehsan** [comment]
Is it okay if I leave these as is? I'm going to remove these files after all of the exp-integration code is ready to go: 

```
components/card-sort/component.js: line 49, col 18, 'category' is already defined.

1 error

components/free-response-form/component.js: line 3, col 30, 't' is defined but never used.

1 error

components/rating-form/component.js: line 53, col 5, '$' is not defined.

1 error

routes/survey.js: line 11, col 7, Missing semicolon.
routes/survey.js: line 106, col 16, 'item' is defined but never used.
```

**Comment by samanehsan** [comment]
Done making those changes @samchrisinger! 🚀 

https://github.com/CenterForOpenScience/exp-addons/pull/105 adds the jam-auth code to exp-addons

**Comment by samchrisinger** [comment]
Waiting on #105 to merge, but lgtm.

**Comment by samanehsan** [comment]
@samchrisinger, I think this should go into a feature branch so that staging isn't broken! It still needs a few things, which I listed above under reviewer notes ☀️

File: app/components/isp-jam-auth/component.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import Ember from 'ember';
+import ExpJamAuth from 'exp-player/components/jam-auth/component';
+
+export default ExpJamAuth.extend({
+  password: 'password'
```
Comment (samchrisinger) [inline app/components/isp-jam-auth/component.js:5]: As a todo for us both, let's discuss better ways to do this.

**Pull request merged**